### PR TITLE
Add MacOSX support without hardware

### LINKED ISSUE #1639: Get it working on Mac without hardware
- [ ] Add MacOSX XOPs: JSON/TUF/ZeroMQ
- [ ] Manual test running Basic.pxp
- [ ] Update manual instructions

### PR DESCRIPTION
**Pull request by t-b**
Close #1639

### CODE REVIEW

File: Packages/doc/installation.rst [hunk 1]
```diff
@@ -195,3 +199,28 @@ hardware related XOPs present.
    -  In ``Igor Help Files`` a shortcut pointing to HelpFiles-IP9
 
 -  Start Igor Pro
+
+MacOSX (without hardware)
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We do support analysis on MacOSX only. We neither support data acquisition nor
```
Comment (timjarsky) [inline Packages/doc/installation.rst:206]: Analysis support only. Data acquisition and NWBv2 export are not supported on MacOSx.

File: README.md [hunk 2]
```diff
@@ -55,14 +55,22 @@ for new hardware.
...
+### Data evaluation
+
+- Windows 10x64 or MacOSX 10.10
+
 ## Getting started
 
 - [Download](https://github.com/AllenInstitute/MIES/releases/tag/latest) the latest release
-- Run the installer
+- Windows: Run the installer
+- MacOSX: [Manual installation](https://alleninstitute.github.io/MIES/installation.html#manual-installation)
```
Comment (timjarsky) [inline README.md:73]: Might be good to say here that MaxOSX is analysis only.

**Comment by t-b** [comment]
Diff:

```diff
 -:  --------- >  1:  d7470dfd2 CI: Rename TestITC job to TestITC18
 -:  --------- >  2:  6f983d27b CI: Add testing with ITC1600
 -:  --------- >  3:  ea371e1d2 DeviceNameGeneratorMD1: Use ITC1600 device zero
 -:  --------- >  4:  ec36e2e11 Tests/UnassociatedChannelsAndTTLs: Fix stimset length test assertion for ITC1600
 -:  --------- >  5:  761a18b8a GetActiveChannelsTTL: Add parantheses around operators of different precedence
 -:  --------- >  6:  a88075499 GetActiveChannelsTTL: Fix TTLbits for TTL rack one
 -:  --------- >  7:  edf99d7d8 EP_GetEpochs/EP_FetchEpochs: Improve documentation
 -:  --------- >  8:  b1271f37f NWB_AppendSweepLowLevel: Fix exporting TTL epoch information for ITC hardware
 -:  --------- >  9:  675690df6 tests/UnassociatedChannelsAndTTLs: Use different channels and stimsets for ITC rack one
 -:  --------- > 10:  bd630527e Packages/IPNWB: Update
 -:  --------- > 11:  0bcb9db47 MIES_MiesUtilities.ipf: Make some more helper routines threadsafe
 -:  --------- > 12:  a884ff57c SplitSweepIntoComponents: Make backup creation optional
 -:  --------- > 13:  f4b5a0f63 NWB_AppendSweepLowLevel: Fix and rework TTL channel saving
 -:  --------- > 14:  ec0213b66 SplitSweepIntoComponents: Workaround IP bug in preemptive thread
 -:  --------- > 15:  7ddaa4419 FindRightMostHighBit: Add it
 -:  --------- > 16:  b7afd53ba MIES_NeuroDataWithoutBorders.ipf: Introduce functions for TTL channel suffix generation in NWB
 -:  --------- > 17:  24d2e848d Fix: Testpulse calculation failed if TP data is NaN
 -:  --------- > 18:  ea4ab90c6 UTF_PA_Tests.ipf: Remove duplicated ModifyBrowser call
 -:  --------- > 19:  c9f939f98 Tests: Only close Databrowser in CI
 -:  --------- > 20:  859042cb0 Format: Add brackets around operation value division
 -:  --------- > 21:  dd6eb9dff Format: Prevent variable name "as"
 -:  --------- > 22:  ade5479eb Format: Add brackets around operation flag arguments
 1:  f6f5899ed = 23:  b2f49fa83 XOPs-MacOSX-IP9-64bit/JSON-64.xop: Add it
 2:  44f958c48 = 24:  087701c07 XOPs-MacOSX-IP9-64bit/TUF-64.xop: Add it
 3:  c6a3442df = 25:  f0551f12f XOPs-MacOSX-IP9-64bit/ZeroMQ-64.xop: Add it
 4:  4725e4234 = 26:  918cdf78c .pre-commit-config.yaml: Ignore MacOSX XOP folder
 5:  727fffd14 ! 27:  a58d2dbfa installation.rst: Document how to manually install on MacOSX
    @@ Packages/doc/installation.rst: hardware related XOPs present.
     +MacOSX (without hardware)
     +~~~~~~~~~~~~~~~~~~~~~~~~~
     +
    -+We do support analysis on MacOSX only. We neither support data acquisition nor
    -+export into NWBv2.
    ++Analysis support only. Data acquisition and NWBv2 export are not supported on MacOSx.
     +
     +-  Quit Igor Pro
    ++-  Get the MIES source code, see
    ++   `here <https://alleninstitute.github.io/MIES/developers.html#latest-development-version-from-main-branch>`__
    ++   for instructions.
     +-  Create the following symlinks in
     +   ``/Users/$username/Documents/WaveMetrics/Igor Pro 9 User Files``
     +
 6:  89975804e = 28:  80b3cfa88 Add support for nightly IP versions on MacOSX
 7:  35ff5d66c ! 29:  956bb5bca Add stubs for MiesUtils XOP functions on MacOSX
    @@ Commit message
         can be conveniently reimplemented with the TUF XOP.
     
      ## Packages/MIES/MIES_Utilities.ipf ##
    -@@ Packages/MIES/MIES_Utilities.ipf: threadsafe Function/S UpperCaseFirstChar(string str)
    +@@ Packages/MIES/MIES_Utilities.ipf: threadsafe Function FindRightMostHighBit(uint64 value)
      
    - 	return UpperStr(str[0]) + str[1, len - 1]
    + 	return NaN
      End
     +
     +#ifdef MACINTOSH
 8:  66752b31c = 30:  ff4a13c7d GetHFSPath: Add support for MacOSX
 9:  4618b8d9c = 31:  6a607523e UTF_AnalysisBrowserTest.ipf: Make them work on MacOSX
10:  6486c2264 = 32:  01dcfafde CreateMiesVersion: Add support for MacOSX
11:  9d2e381d9 = 33:  01948807a Tests/FR_FolderExistsWorks/FR_FileExistsWorks: Make it work on MacOSX
12:  9b618f8e6 = 34:  b37c95b51 DAP_GetITCDeviceList/DAP_GetNIDeviceList: Make them failsafe on MacOSX
13:  783e3d4fd = 35:  deb148109 MEN_CreateIssueOnGithub: Workaround issue on MacOSX
14:  d5637922f = 36:  82cb0f37e Tests/CheckCompilation_IGNORE: Make it pass on MacOSX
15:  a89b8f09c = 37:  dd9a02786 CHI_CheckInstallation: Add preliminary support for MacOSX
16:  9f35ff17f = 38:  03b7be7fb GetDrive: Make it work on Macintosh
17:  d4656e3e7 ! 39:  ff48c8c98 README.md: Mention MacOSX support
    @@ README.md: for new hardware.
      - [Download](https://github.com/AllenInstitute/MIES/releases/tag/latest) the latest release
     -- Run the installer
     +- Windows: Run the installer
    -+- MacOSX: [Manual installation](https://alleninstitute.github.io/MIES/installation.html#manual-installation)
    ++- MacOSX (Analysis only): [Manual installation](https://alleninstitute.github.io/MIES/installation.html#manual-installation)
      - [View](https://alleninstitute.github.io/MIES/user.html) the documentation
      
      ## Support statement
```

File: README.md (hunk 2 above)
Comment (t-b) [reply README.md]: Fixed.

File: Packages/doc/installation.rst (hunk 1 above)
Comment (t-b) [reply Packages/doc/installation.rst]: Fixed.

**Comment by t-b** [comment]
Will merge once CI passes.

**Pull request merged**